cobalt/testing/filters/linux-x64x11/base_unittests_filter.json

```diff
@@ -102,6 +102,7 @@
     "ProcessUtilTest.FDRemappingIncludesStdio",
     "ImmediateCrashTest.ExpectedOpcodeSequence",
     "PostTaskAndReplyImplTest.PostTaskToStoppedTaskRunnerWithoutSequencedContext",
-    "ThreadGroupImplBlockingTest.MayBlockIncreaseCapacityNestedWillBlock"
+    "ThreadGroupImplBlockingTest.MayBlockIncreaseCapacityNestedWillBlock",
+    "ElfReaderTestWithCurrentImage.ReadElfBuildId"
```

Comment:
I assume this is disabled because it relies on base functionality you disabled, in which case you should prefix the actual test name(s) with DISABLED_ in the source
Alternatively, if these changes make an entire suite of tests obsolete, remove the source in the GN file